service/src/test/java/skills/intTests/UserPointsSpecs.groovy

```diff
@@ -1164,4 +1164,126 @@ class UserPointsSpecs extends DefaultIntSpec {
         data4.userTag == ['tag1', 'tag2', 'tag4', 'tag5']
     }
 
+    def 'user project points maximum filter is exclusive' () {
```

Comment:
Please enhance the test to include two projects with both distinct and overlapping users.